bin/copyRawDataToPrm.sh

```diff
@@ -568,6 +563,16 @@ else
 		mkdir -m 2750 -p "${PRM_ROOT_DIR}/Samplesheets/"
 		# shellcheck disable=SC2174
 		mkdir -m 2750 -p "${PRM_ROOT_DIR}/Samplesheets/archive/"
+		
```

Comment:
Deze verplaatsing snap ik niet. Nu wordt er altijd een nieuwe *.started gemaakt en daarna pas gecontroleerd of er niet al een *.finished is en deze run overgeslagen moet worden.

Reply:
ik denk dat ik hier net wat teveel heb verplaatst, het schrijven van een logfile voordat de logfolder is gemaakt was het probleem, vandaar dat ik daar wat ging schuiven..
Update komt er aan.

Reply:
Ok, proberen te loggen voordat de log folder er is wil inderdaad niet. De volgorde is nu beter, maar ik snap nog niet waarom er een
printf "finished" > "${JOB_CONTROLE_FILE_BASE}.trace_putFromFile_setProcessRawData.csv" 
geschreven wordt vlak voordat
printf '' > "${JOB_CONTROLE_FILE_BASE}.started"
een lege *.started maakt of een reeds bestaande reset naar een lege file. Moet die finished voor de "${JOB_CONTROLE_FILE_BASE}.trace_putFromFile_setProcessRawData.csv" niet later?

Reply:
de processRawData is de kolom procesRawData (dus demultiplexing/arrayConversion) in de overview entiteit op de molgenis server
Het updaten van die kolom van started naar finished kan niet worden gedaan door de automated aangezien de demultiplexing word uitgevoerd en vervolgens de jobs submit (en dan is de demultiplexing klaar).
Ik dacht als ik nu die stap in de copyRawDataToPrm doe heb ik dat probleem ook weer opgelost.

Reply:
Ok, dus als ik het goed begrijp probeer je de status van een vorige stap op finished te zetten als copyRawDataToPrm begint, correct? Dan lijkt me dat Ok. Het is alleen wat verwarrend als je de code de eerste keer leest, dus wellicht is een comment, die uitlegt dat het hier om een vorige stap gaat op z'n plaats...